GraalVM Native Image Lab 14 - Error Build Native Image

https://graemerocher.github.io/micronaut-oraclecloud-workshop/workshop/?lab=lab-14-building-native-image#BuildingaNativeImagewithGradle
```
[opc@micronaut-hol example]$ ./gradlew nativeImage --stacktrace
> Task :nativeImage FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':nativeImage'.
> A problem occurred starting process 'command 'native-image''
* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':nativeImage'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:200)
        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:263)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:198)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:179)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:109)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'native-image'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.startProcess(ExecHandleRunner.java:98)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:71)
        ... 4 more
Caused by: java.io.IOException: Cannot run program "native-image" (in directory "/home/opc/example/build/native-image"): error=2, No such file or directory
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
```

Native Image version
```
[opc@micronaut-hol example]$ native-image --version
GraalVM Version 21.0.0 (Java Version 11.0.10+8-jvmci-21.0-b06)
[opc@micronaut-hol example]$ echo $JAVA_HOME
/usr/lib64/graalvm/graalvm-ce-java11-21.0.0
[opc@micronaut-hol example]$ ls /usr/lib64/graalvm/graalvm-ce-java11-21.0.0/bin
gu         java     javap     jdb        jfr     jinfo  jmap  jrunscript  jstack  jvisualvm  native-image  npx       rebuild-images  rmiregistry jar  javac    jcmd      jdeprscan  jhsdb   jjs    jmod  js          jstat   keytool    node          pack200   rmic            serialver jarsigner  javadoc  jconsole  jdeps      jimage  jlink  jps   jshell      jstatd  lli    npm           polyglot  rmid            unpack200
```
Debug Mode
```
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :nativeImage failed
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :nativeImage completed, executed: true
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Execution worker for ':' Thread 2: released lock on :
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.internal.work.DefaultWorkerLeaseService] Worker lease root.1.16 completed (1 worker(s) in use)
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Execution worker for ':' Thread 2: released lock on root.1.16
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Execution worker for ':' Thread 2,5,main]] finished, busy: 0.017 secs, idle: 4.507 secs
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Execution worker for ':',5,main]] finished, busy: 4.487 secs, idle: 0.042 secs
2021-02-10T10:46:15.951+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Execution worker for ':' Thread 3,5,main]] finished, busy: 0.0 secs, idle: 4.496 secs
2021-02-10T10:46:15.952+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Daemon worker Thread 12,5,main]] finished, busy: 0.0 secs, idle: 4.515 secs
2021-02-10T10:46:15.953+0000 [DEBUG] [org.gradle.execution.taskgraph.DefaultTaskExecutionGraph] Timing: Executing the DAG took 4.564 secs
2021-02-10T10:46:15.955+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Run tasks'
2021-02-10T10:46:15.955+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Run tasks' completed
2021-02-10T10:46:15.962+0000 [DEBUG] [org.gradle.internal.work.DefaultWorkerLeaseService] Worker lease root.1 completed (0 worker(s) in use)
2021-02-10T10:46:15.962+0000 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Daemon worker Thread 12: released lock on root.1
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Execution failed for task ':nativeImage'.
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > A problem occurred starting process 'command 'native-image''
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
2021-02-10T10:46:15.962+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.
2021-02-10T10:46:15.963+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]
2021-02-10T10:46:15.963+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org
2021-02-10T10:46:15.963+0000 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger]
2021-02-10T10:46:15.963+0000 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 6s
2021-02-10T10:46:15.963+0000 [LIFECYCLE] [org.gradle.internal.buildevents.TaskExecutionStatisticsReporter] 3 actionable tasks: 2 executed, 1 up-to-date
2021-02-10T10:46:15.963+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Build finished for file system watching' started
2021-02-10T10:46:15.963+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Build finished for file system watching'
2021-02-10T10:46:15.963+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Build finished for file system watching' completed
2021-02-10T10:46:15.963+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Run build'
2021-02-10T10:46:15.963+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Run build' completed
2021-02-10T10:46:15.966+0000 [DEBUG] [org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess] Releasing file lock for cache directory md-supplier (/home/opc/.gradle/caches/6.7/md-supplier)
2021-02-10T10:46:15.966+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on cache directory md-supplier (/home/opc/.gradle/caches/6.7/md-supplier).
2021-02-10T10:46:15.967+0000 [DEBUG] [org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess] Releasing file lock for Build Output Cleanup Cache (/home/opc/example/.gradle/buildOutputCleanup)
```

--- a/introduction/prerequisites.md
+++ b/introduction/prerequisites.md
@@ -27,7 +27,7 @@ Once you have your Oracle Cloud account ready, click the following button:
 [![Create Oracle Cloud Stack][magic_button]][magic_jidea_terraform_stack]
 
 [magic_button]: https://oci-resourcemanager-plugin.plugins.oci.oraclecloud.com/latest/deploy-to-oracle-cloud.svg
-[magic_jidea_terraform_stack]: https://cloud.oracle.com/resourcemanager/stacks/create?zipUrl=https://objectstorage.us-ashburn-1.oraclecloud.com/n/cloudnative-devrel/b/micronaut-hol/o/terraform%2Fmicronaut-hol-with-projector-v2.zip
+[magic_jidea_terraform_stack]: https://cloud.oracle.com/resourcemanager/stacks/create?zipUrl=https://objectstorage.us-ashburn-1.oraclecloud.com/n/cloudnative-devrel/b/micronaut-hol/o/terraform%2Fmicronaut-hol-with-projector-v3.zip
 
 Clicking the button will take you to Oracle Cloud Resource Manager where you will use a pre-prepared Terraform stack to initialize the infrastructure for the lab.
 

